docs/quickstart/beginner.rst

```diff
@@ -36,29 +37,37 @@ flow without relying on external files for the input and the expected results
 
 Input Data
 ----------
-Most pygrametl abstractions either produce, consume, or operate on data in rows.
-A row is a Python :class:`.dict` where the row's column names are the keys and
-the values are the data the row contains. For more information about the data
-source provided by pygrametl see :ref:`datasources`.
+Most pygrametl abstractions either produce, consume, or operate on data in
+rows. A row is a Python :class:`.dict` where the row's column names are the
+keys and the values are the data the row contains. For more information about
+the data source provided by pygrametl see :ref:`datasources`.
 
 The most important data for the warehouse are which books have been sold. This
-information can be extracted from the chain's sales records. The table storing
-this data is shown below:
+information can be extracted from the book stores' sales records which are
+stored in the SQLite database :formatref:`sale.sqlite
+<../_static/sale.sqlite>`. Of course, storing sales records from multiple book
+stores is not a common use-case for SQLite. However, SQLite is used in this
+beginner guide as it makes sharing the input data simple and demonstrates
+pygrametl's ability to read and write from any RDBMS that provides a :pep:`249`
+connection. The data is stored in the table sale as shown below:
 
 .. code-block:: none
 
-   | book:text             | genre:text | store:text | timestamp:date | sale:int |
+   | book:text             | genre:text | store:text | date:date      | sale:int |
    | --------------------- | ---------- | ---------- | -------------- | -------- |
    | Nineteen Eighty-Four  | Novel      | Aalborg    | 2005/08/05     | 50       |
    | Calvin and Hobbes One | Comic      | Aalborg    | 2005/08/05     | 25       |
    | The Silver Spoon      | Cookbook   | Aalborg    | 2005/08/14     | 5        |
    | The Silver Spoon      | Cookbook   | Odense     | 2005/09/01     | 7        |
-   | ....                  |            |            |                |		 |
+   | ....                  |            |            |                |          |
 
 
-As the geographical information stored in the sales records are limited, the
-location dimension is pre-filled with data from the CSV file region.csv. The
-file contains data as shown below (tabs are added for readability):
+The book titles and genres are extracted from the `CMU Book Summary Dataset
+<https://www.cs.cmu.edu/~dbamman/booksummaries.html>`_. As the geographical
+information stored in the sales records is limited, the location dimension must
+be pre-filled with data from the CSV file :formatref:`region.csv
```

Comment:
Could we make a single zip file with all the needed data in addition to the full ETL program? It would then be very easy to users to try out

Reply:
Fixed with commit 190f1d0